modules/bene_core/src/Plugin/Block/SubNavigationBlock.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Drupal\bene_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Link;
+
+/**
+ * Provides a 'SubNavigationBlock' block.
+ *
+ * @Block(
+ *  id = "bene_sub_navigation_block",
+ *  admin_label = @Translation("Bene sub navigation"),
+ *  category = @Translation("Bene")
+ * )
+ */
+class SubNavigationBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return [] + parent::defaultConfiguration();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) {
+    $form = [];
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockSubmit($form, FormStateInterface $form_state) {
+
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    // Disable block cache as page title will change per-page.
+    $build = [
+      '#cache' => [
```

Comment:
Caching could be enabled using a combo of cache tags (for each item in the breadcrumb as well as the current page) and contexts (route). See https://drupal.stackexchange.com/questions/199527/how-do-i-correctly-setup-caching-for-my-custom-block-showing-content-depending-o for some ideas.

This is low priority - UNLESS it's bubbling up to prevent caching of the page as a whole.

Reply:
This isn't affecting page caching so I've flagged it for later review. I'm moving on to #67 so we can hand this over for theming because @heypaxton has time this week.